cli/pack/common.go

```diff
@@ -277,14 +277,30 @@ func getDestAppDir(bundleEnvPath, appName string,
 
 // copyApplications copies applications from current env to the result bundle.
 func copyApplications(bundleEnvPath string, packCtx *PackCtx,
-	cliOpts, newOpts *config.CliOpts) error {
-	var err error
+	cliOpts, newOpts *config.CliOpts, toIgnore map[string]struct{}) error {
```

Comment:
The name of this function is "copy applications". One of the arguments is a set of paths to do not copy. Do we exclude the same paths for all applications? If so, this is not what was expected.

Reply:
Changed the implementation, now files are deleted from the directory after full copying by a separate function 'removeIgnoredFiles'